org.desmume.DeSmuME.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.desmume.DeSmuME</id>
+  <name>DeSmuME</name>
+  <summary>Nintendo DS emulator</summary>
+  <project_license>GPL-2.0-or-later</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <developer_name>DeSmuME Team</developer_name>
+  <icon type="remote">https://linkmauve.fr/files/desmume-logo.svg</icon>
```

Comment:
if you're already shipping an icon with the app, there's no need to add one unless you want it as a fallback?